crates/rolldown_ecmascript_utils/src/ast_snippet.rs

```diff
@@ -158,7 +146,13 @@ impl<'ast> AstSnippet<'ast> {
     arg: ast::Expression<'ast>,
   ) -> ast::Expression<'ast> {
     let arg = ast::Argument::from(arg);
-    let mut call_expr = self.call_expr(name);
+    let mut call_expr = self.builder.call_expression(
+      SPAN,
+      self.builder.expression_identifier(SPAN, self.builder.atom(name)),
+      NONE,
+      self.builder.vec(),
```

Comment:
A small note:

`AstBuilder::vec1` can be used for creating a `Vec` with a single entry. It's shorter then creating a `Vec` and then pushing to it, and also slightly more performant.

Incidentally, there's also `AstBuilder::vec_from_array` if you want to create a `Vec` with multiple entries e.g. `builder.vec_from_array([node1, node2])`. Again, that's a little more performant.